feat: added lab to build a quickstart (#1686)
newrelic/developer-website: src/markdown-pages/instant-observability/build-a-quickstart/index.mdx

title: 'Build a quickstart'
template: 'GuideTemplate'
description: 'New Relic One quickstarts gives you a jump start in your observability journey with New Relic and allow you to quickly and easily observe and monitor your services'
---

You’ve developed a new database called flashDB, the lightest and fastest database in the space. Your database is already very popular among the developers. Now, you want to help users of flashDB to quickly and easily observe it in New Relic.

![build a quickstart](../../../images/instant-observability/lab.png)

To achieve your goal, contribute a quickstart to New Relic I/O so other users can quickly monitor flashDB.

## Why quickstarts?

New Relic allows you to collect, monitor, and report data from different sources. While we offer a large variety of open-source integrations for telemetry tools, you can also use Telemetry SDKs to create your own data solutions. You can use dashboards to view, explore, customize, understand, and correlate the data you collect using different user-friendly charts. 
You can also set alert policies to monitor key performance metrics and receive notification for anomalies. 

> Often, you had to custom build these solutions to observe your applications in New Relic but now, we offer "quickstarts" to help you and others quickly and easily observe your applications in New Relic. 

Reviewer: ```suggestion
Traditionally, FlashDB users would have to spend crucial development hours creating dashboards, alerts, and other entities to monitor their database. But with quickstarts, you can do that work for them.

Quickstarts provide immediate value to your users. They provide:

 - Clear instructions for instrumenting your services
 - Observability building blocks, like dashboards and alerts, pre-packaged for easy installation
```

New Relic One quickstarts provide immediate value for your specific use case. They include:
- Clear instructions for instrumenting your services
- Observbility building blocks like dashboards and alerts

All of this is available through an open ecosystem where New Relic developers, partners, and customers contribute their best-practice solutions. 

> Since you're the developer of flashDB, you can also contribute your best-practice solution to help the users of flashDB quickly monitor it in New Relic.

Reviewer: ```suggestion
You, as the developer of FlashDB, are a great candidate to contribute a quickstart to help your users of quickly monitor their instances.
```

## Learning Objectives

> In this lab, you will . 

Reviewer: ```suggestion
In this lab, you will:
```

- Create a dashboard
- Create alerts 
- Create a quickstart

>     - Package up your dashboard & alerts into quickstart

Reviewer: ```suggestion
    - Package up your dashboard & alerts
```

>     - Add quickstart description

Reviewer: ```suggestion
    - Add a description
```

>     - Contribute quickstart to GitHub

Reviewer: ```suggestion
    - Contribute it to GitHub
```

## Prerequisite

> - New Relic One account

Reviewer: ```suggestion
- Free [New Relic account](https://login.newrelic.com/login)
```
